### PR TITLE
[Auctions] User can view own listings, landing page displays open auctions

### PR DESCRIPTION
**Pull request by purnima-nyc**
## Description
- User can view own listings by clicking on 'My Listings' in the nav bar
- Landing page displays open auctions only

https://trello.com/c/kL3l2Tyi
https://trello.com/c/FB1IiVk7

### CODE REVIEW

File: lib/ebaze/auctions.ex [hunk 1]
```diff
@@ -8,6 +8,23 @@ defmodule Ebaze.Auctions do
     Repo.all(Auction)
   end
 
+  def list_my_auctions(user) do
+    Repo.all(
+      from auction in Auction,
```
Comment (ssunday) [inline lib/ebaze/auctions.ex]: Could you refactor this so the logic for listing out all actions is extracted out from the where query by using piping?

File: lib/ebaze_web/router.ex [hunk 2]
```diff
@@ -42,6 +42,8 @@ defmodule EbazeWeb.Router do
 
     resources "/auctions", AuctionController, except: [:index]
 
+    get "/sell", AuctionController, :sell
```
Comment (ssunday) [inline lib/ebaze_web/router.ex]: Is this the proper name for this route? Should it be `my-auctions` or `/profile/auctions`?

File: lib/ebaze_web/templates/auction/sell.html.eex [hunk 3]
```diff
@@ -0,0 +1,40 @@
+<h1>My Auctions</h1>
+
+<%= if length(@auctions) != 0 do %> 
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Description</th>
+      <th>Sold status</th>
```
Comment (ssunday) [inline lib/ebaze_web/templates/auction/sell.html.eex]: Can the table for auctions be yanked out into a partial?

File: lib/ebaze/auctions.ex (hunk 1 above)
Comment (purnima-nyc) [reply lib/ebaze/auctions.ex]: @ssunday is`from(auction in Auction, where: auction.created_by_id == ^user.id)
    |> Repo.all()` good?

Comment (ssunday) [reply lib/ebaze/auctions.ex]: Could you extract out `from(c in Auction)` and then pipe the where to it?

Comment (purnima-nyc) [reply lib/ebaze/auctions.ex]: `Auction
    |> where([auction], auction.created_by_id == ^user.id)
    |> Repo.all()`

Comment (ssunday) [reply lib/ebaze/auctions.ex]: I think it can be `where(created_by_id: ^user.id)`, but yes!

Comment (purnima-nyc) [reply lib/ebaze/auctions.ex]: thats super cool!

File: lib/ebaze/auctions.ex [hunk 4]
```diff
@@ -8,6 +8,21 @@ defmodule Ebaze.Auctions do
     Repo.all(Auction)
   end
 
+  def list_my_auctions(user) do
+    Auction
```
Comment (ssunday) [inline lib/ebaze/auctions.ex]: `from(auction in Auction)`

File: lib/ebaze/auctions.ex [hunk 5]
```diff
@@ -8,6 +8,21 @@ defmodule Ebaze.Auctions do
...
+    from(auction in Auction)
+    |> where(created_by_id: ^user.id)
+    |> Repo.all()
+  end
+
+  def list_open_auctions() do
+    now = DateTime.utc_now()
+    not_specified = nil
+
+    from(auction in Auction)
```
Comment (ssunday) [inline lib/ebaze/auctions.ex]: private function for `from(auction in Auction) to dry it up

**Comment by purnima-nyc** [comment]
Landing page
Screen Shot 2019-09-25 at 3 03 46 PM

User can view own listings
Screen Shot 2019-09-25 at 3 04 04 PM

User with no listings
Screen Shot 2019-09-25 at 3 04 23 PM

**Pull request merged**